ViewModels/Cart.cs

```diff
@@ -9,7 +9,10 @@ namespace BangazonWeb.ViewModels
   public class CartView : BaseViewModel
   {
     public float TotalPrice { get; set; }
+    public Order Order { get; set; }
+    public PaymentType PaymentType { get; set; }
     public IEnumerable<Product> ActiveProducts { get; set; }
+    public IEnumerable<Product> LineItems { get; set; }
```

Comment:
It has to do with the different ways that the line items are being queried from the database. When the order is referring to an incomplete order they are active products, when they are being completed at that point the products are line items.